submissions/egekenzeKelechi/html-forms/contact.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact ::</title>
+</head>
+<body>
+    <div class="container">
+        <nav>
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="../html/products.html">Product</a></li>
+                <li><a href="contact.html">Contact</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+
+        <br><br><br><br><br>
+        <h3>Want more information, fill the form below and a representative will get back to you</h3>
+        <br><br><br>
+
+        <h5>Please fill all the details in asterisks.</h5>
+        <form action="#" method="post">
```

Comment:
@Keicee32 Remove the attributes for `action` and `method` attributes, It is not needed yet.